backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -232,21 +232,6 @@ public async Task<ActionResult<int>> UpdateLexEntryCount(string code)
         return result is null ? NotFound() : result;
     }
 
-    [HttpPost("updateAllLexEntryCounts")]
-    [AdminRequired]
-    public async Task<ActionResult<int>> UpdateAllLexEntryCounts(bool onlyUnknown = true, int limit = 100, int delayMs = 10)
```

Comment:
we don't need to remove the API if we still think it's useful, but that's up to you

Reply:
I don't see a situation where it would be useful, so I'd rather have a smaller API surface. If it turns out we need it in the future, it'll be quite easy to find this PR and copy the code to put it back.